app/src/main/java/mobile/programming/musicApp/service/InstrumentService.kt

```diff
@@ -1,4 +1,4 @@
-package mobile.programming.musicapp.service
+package mobile.programming.musicApp.service
```

Comment:
Here, I have also changed the multi-name package name to camelcase.

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase